packages/plugin-ext/src/plugin/known-commands.ts

```diff
@@ -294,6 +294,7 @@ export namespace KnownCommands {
     mappings['vscode.prepareCallHierarchy'] = ['vscode.prepareCallHierarchy', CONVERT_VSCODE_TO_MONACO, CONVERT_MONACO_TO_VSCODE];
     mappings['vscode.provideIncomingCalls'] = ['vscode.provideIncomingCalls', CONVERT_VSCODE_TO_MONACO, CONVERT_MONACO_TO_VSCODE];
     mappings['vscode.provideOutgoingCalls'] = ['vscode.provideOutgoingCalls', CONVERT_VSCODE_TO_MONACO, CONVERT_MONACO_TO_VSCODE];
+    mappings['vscode.open'] = ['vscode.open', CONVERT_VSCODE_TO_MONACO];
```

Comment:
Strictly speaking, this isn't necessary to get the version of the plugin downloaded by our scripts to work because it uses `vscode.window.showTextDocument` rather than this command. However, later versions of the plugin use the `vscode.open` command, and we have an off-by-one issue when handling `TextDocumentShowOptions` passed by plugins into that command.